### PR TITLE
Implement and save guestbook comments

### PR DESCRIPTION
**Pull request by shulijones**
Walkthrough steps #5 and 6.

https://shulijones-step-2020.appspot.com/index.html

### CODE REVIEW

**Review by kyracq — APPROVED** [review]
LGTM!

**Review by nsri-google — DISMISSED** [review]
Please make the suggested changes and submit again for review.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,57 @@
...
+    
+    Gson gson = new Gson();
+    response.getWriter().println(gson.toJson(comments));
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long timestamp = System.currentTimeMillis();
+    String comment = request.getParameter("text-input");
+    //comments.add(comment);
```
Comment (nsri-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: A common software engineering "do not do" is to leave commented out code. Delete if you don't need this.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,57 @@
...
+
+    response.setContentType("text/html");
+    Gson gson = new Gson();
```
Comment (nsri-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Do you need gson? It is not used anywhere.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -19,14 +19,57 @@
...
+    //response.getWriter().println(gson.toJson(comments));
```
Comment (nsri-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Delete commented out code.

**Review by nsri-google — CHANGES_REQUESTED** [review]

**Review by nsri-google — APPROVED** [review]

**Review by anicr7 — APPROVED** [review]

**Pull request merged**